src/locales/ko/pokemon-summary.ts

```diff
@@ -0,0 +1,17 @@
+import { SimpleTranslationEntries } from "#app/plugins/i18n";
+
+export const pokemonSummary: SimpleTranslationEntries = {
+  "pokemonInfo": "Pokémon Info",
+  "status": "Status",
+  "powerAccuracyCategory": "Power\nAccuracy\nCategory",
+  "type": "Type",
+  "unknownTrainer": "Unknown",
+  "ot": "OT",
+  "luck": "Luck",
+  "nature": "nature",
+  "apparently": "apparently",
+  "metAtLv": "met at Lv.",
+  "expPoints": "Exp. Points",
+  "nextLv": "Next Lv.",
+  "cancel": "Cancel",
```

Comment:
```suggestion
  "pokemonInfo": "포켓몬 정보",
  "status": "능력치",
  "powerAccuracyCategory": "위력\n명중\n분류",
  "type": "타입",
  "unknownTrainer": "알수없음",
  "ot": "어버이",
  "luck": "행운",
  "nature": "성격",
  "apparently": "apparently",
  "metAtLv": "met at Lv.",
  "expPoints": "현재 경험치",
  "nextLv": "다음 레벨까지",
  "cancel": "그만둔다",
```

Due to differences in word order between Korean and English, those parts have not been translated.

Reply:
"status"는 9세대 스칼렛/바이올렛 기준으로 "스테이터스"라고 되어 있어 "능력치"보다는 "스테이터스"가 좋을 듯합니다. 마찬가지로 "cancel"도 "돌아간다"로 되어 있습니다.

Reply:
먼저 올리신 걸 나중에 봤네요.
의견주신 것들 고치면 거의 일치하니까 
제안은 취소할께요